usr/share/rear/lib/layout-functions.sh

```diff
@@ -806,17 +833,17 @@ blkid_label_of_device() {
     echo "$label"
 }
 
-# Returns 1 if the device is an LVM physical volume
-# Returns 0 otherwise or if the device doesn't exists
+# Returns true if the device is an LVM physical volume
+# Returns false otherwise or if the device doesn't exists
```

Comment:
@pcahyna 
thank you for cleaning up the inverse logic
of the is_disk_a_pv return code.

Reply:
An addedum regarding is_disk_a_pv:
I was wondering about the word 'disk' in this function
because a LVM PV can be a disk or a partition
(in general a PV is a block device, e.g. also a LUKS volume).
The is_disk_a_pv function was introduced via
https://github.com/rear/rear/pull/901
so the is_disk_a_pv function is meant to be used
only with a disk as argument when installing GRUB[2].